modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -19,9 +19,11 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
+
 		else {
 			ObjectAdmin.deleteAllCustomObjectsViaAPI();
 		}
+
```

Comment:
This looks like it was caught by SF? If you need to make SF changes to code that wasn't part of your changes, you can make them in a separate commit with the message "LPS-156807 Auto SF".